chromium_src/ios/chrome/browser/application_context_impl.mm

```diff
@@ -364,3 +367,17 @@ void BindNetworkChangeManagerReceiver(
 void ApplicationContextImpl::CreateGCMDriver() {
   DCHECK(thread_checker_.CalledOnValidThread());
 }
+
+PromosManager* ApplicationContextImpl::GetPromosManager() {
```

Comment:
There are no usages of this `PromosManager` right? The usages I found at least in Chromium do not check for `nullptr` before usage.

Reply:
@cdesouza-chromium I think we should copy the implementation of `GetPromosManager` from https://chromium-review.googlesource.com/c/chromium/src/+/3785128, but let me know what you think.

Reply:
Fixed in b97103445cda52812f7b2816e1d49f95b91adbc4.